### PR TITLE
Add dot parameter description to buck query docs.

### PR DESCRIPTION
**Pull request by mogers**
Summary:
Add dot parameter description to buck query docs, and provide sample usage.
Fix related wording in the 'allpaths' operator.

Test Plan:
Publish the docs locally and inspect the generated page.
  $ bash docs/soyweb-local.sh

### CODE REVIEW

**Comment by mogers** [comment]
I think I had this at some point (at least the allpaths wording fix) but maybe messed it up when splitting diffs.

File: docs/command/query.soy [hunk 1]
```diff
@@ -321,14 +321,14 @@ the repository and the target <code>//foo:bar</code>.
...
 {sp}parameter to generate an image. Sample usage:
 </p>
 
-<p><pre>
+<p><pre class="prettyprint lang-py">
 {literal}
-buck query "allpaths(//foo:bar, //foo/bar/lib:baz)" --dot > result.dot
-dot -Tpng result.dot -o image.png
+$ buck query "allpaths(//foo:bar, //foo/bar/lib:baz)" --dot > result.dot
+$ dot -Tpng result.dot -o image.png
```
Comment (sdwilsh) [inline docs/command/query.soy]: Why are we using the `$` here and not below?

Comment (mogers) [inline docs/command/query.soy]: You're right, it's inconsistent. Should I change this in all 'buck query' calls in this page?

Comment (sdwilsh) [inline docs/command/query.soy]: Yeah; pick something for this change, and then do another PR to clean it up?

Comment (mogers) [inline docs/command/query.soy]: Sorry for the delay. I added the $ below. 

It's my first upstream PR. I had some conflicts and ended up with 5 commits. Do I need to merge them into one?

Comment (sdwilsh) [inline docs/command/query.soy]: No need to merge.  I can squash later.
________________________________

**Comment by Coneko** [comment]
The lack of stickers in github comments prevents me for expressing my thoughts!

**Comment by mogers** [comment]
@Coneko , feature request? :)
